pessimistic-concurrency/README.md

```diff
@@ -0,0 +1,149 @@
+## Intent
```

Comment:
The yaml frontmatter is missing, see https://github.com/iluwatar/java-design-patterns/wiki/01.-How-to-contribute